lease/lessor.go

```diff
@@ -480,15 +480,13 @@ func (le *lessor) runLoop() {
 
 		le.mu.Lock()
 		if le.isPrimary() {
-			ls = le.findExpiredLeases()
+			// rate limit
+			revokeLimit := leaseRevokeRate / 2
```

Comment:
Move lines 483,484 to line 477 (out of mutex)?

Reply:
or just have `ls = le.findExpiredLeases(leaseRevokeRate / 2)`